src/main/java/frc/team2412/robot/commands/autonomous/JackFiveBallAutoCommand.java

```diff
@@ -93,15 +94,20 @@ public JackFiveBallAutoCommand(DrivebaseSubsystem drivebaseSubsystem, IntakeSubs
 
         ShooterTargetCommand.TurretManager manager = new ShooterTargetCommand.TurretManager(shooterSubsystem,
                 localizer);
+
+        indexSubsystem.setDefaultCommand(new IndexCommand(indexSubsystem, intakeSubsystem));
```

Comment:
This like maybe don't work, need to test again

Reply:
We've already discussed how `CommandGroup`s require every subsystem their component commands require, which means that the default command doesn't run.